addons/resource.language.en_gb/resources/strings.po

```diff
@@ -5247,7 +5247,7 @@ msgstr ""
 #: xbmc\video\dialogs\GUIDialogAudioSubtitleSettings.cpp
 #: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
 msgctxt "#12376"
-msgid "Set as default for all videos"
+msgid "Set as default for all files"
```

Comment:
Every time you press that you save not just what you playing but everything else you have stored.

Streamed media is neither here nor there, I only care about items in my library which are affected by this ALL option.